### PR TITLE
Updated envoy-build-ubuntu image tag

### PR DESCRIPTION
**Pull request by dmitri-d**
* Updated envoy-build-ubuntu image tag
* Updated envoy-build-tools

Signed-off-by: Dmitri Dolguikh <ddolguik@redhat.com>

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Additional Description:
Risk Level: Low
Testing: N/A
Docs Changes: N/A
Release Notes: N/A

### CODE REVIEW

**Comment by dmitri-d** [comment]
@lizan: please take a look when you get a chance.

File: bazel/repository_locations.bzl [hunk 1]
```diff
@@ -67,10 +67,10 @@ DEPENDENCY_REPOSITORIES = dict(
         use_category = ["build"],
     ),
     envoy_build_tools = dict(
-        sha256 = "ff890c70d60e51c7ee80874f85c3905718b7f6929a6c367c850cdd0b9c01d44d",
-        strip_prefix = "envoy-build-tools-efaecf11d76b86551cf42e2354274ac2acd7042f",
+        sha256 = "5dd2ee5dfcf36a47a795df75c2d8b9822dfa7b5c06d4981adc626fce29d9af4e",
+        strip_prefix = "envoy-build-tools-6b903b8e583d346860de002d84b606d04898d637",
         # 2020-06-16
```
Comment (lizan) [inline bazel/repository_locations.bzl:72]: update this line

File: bazel/repository_locations.bzl [hunk 2]
```diff
@@ -67,10 +67,10 @@ DEPENDENCY_REPOSITORIES = dict(
...
-        urls = ["https://github.com/envoyproxy/envoy-build-tools/archive/efaecf11d76b86551cf42e2354274ac2acd7042f.tar.gz"],
+        urls = ["https://github.com/envoyproxy/envoy-build-tools/archive/6b903b8e583d346860de002d84b606d04898d637.tar.gz"],
```
Comment (lizan) [inline bazel/repository_locations.bzl:73]: this should be latest master which include windows toolchain generation as well, https://github.com/envoyproxy/envoy-build-tools/commit/289a5ca65aefd5a76f18f103d1425cfec5591417

Comment (dmitri-d) [reply bazel/repository_locations.bzl]: I was too quick. Updated.

**Comment by dmitri-d** [comment]
* responded to feedback

**Pull request merged**